src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevService.kt

```diff
@@ -1,34 +1,66 @@
 package no.nav.familie.ks.sak.kjerne.brev
 
 import no.nav.familie.kontrakter.felles.arbeidsfordeling.Enhet
+import no.nav.familie.kontrakter.felles.dokarkiv.v2.Dokument
+import no.nav.familie.kontrakter.felles.dokarkiv.v2.Filtype
+import no.nav.familie.kontrakter.felles.dokarkiv.v2.Førsteside
 import no.nav.familie.ks.sak.api.dto.ManueltBrevDto
 import no.nav.familie.ks.sak.api.dto.tilBrev
 import no.nav.familie.ks.sak.common.exception.Feil
+import no.nav.familie.ks.sak.integrasjon.journalføring.UtgåendeJournalføringService
+import no.nav.familie.ks.sak.integrasjon.journalføring.UtgåendeJournalføringService.Companion.DEFAULT_JOURNALFØRENDE_ENHET
+import no.nav.familie.ks.sak.integrasjon.journalføring.domene.DbJournalpost
+import no.nav.familie.ks.sak.integrasjon.journalføring.domene.DbJournalpostType
+import no.nav.familie.ks.sak.integrasjon.journalføring.domene.JournalføringRepository
 import no.nav.familie.ks.sak.kjerne.arbeidsfordeling.ArbeidsfordelingService
+import no.nav.familie.ks.sak.kjerne.behandling.domene.Behandling
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingRepository
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.VilkårsvurderingService
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.AnnenVurderingType
+import no.nav.familie.ks.sak.kjerne.brev.domene.maler.Brevmal
+import no.nav.familie.ks.sak.kjerne.fagsak.domene.FagsakRepository
 import no.nav.familie.ks.sak.kjerne.personopplysninggrunnlag.PersonopplysningGrunnlagService
 import org.springframework.http.HttpStatus
 import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
 
 @Service
 class BrevService(
     private val brevKlient: BrevKlient,
     private val personopplysningGrunnlagService: PersonopplysningGrunnlagService,
-    private val arbeidsfordelingService: ArbeidsfordelingService
+    private val arbeidsfordelingService: ArbeidsfordelingService,
+    private val utgåendeJournalføringService: UtgåendeJournalføringService,
+    private val vilkårsvurderingService: VilkårsvurderingService,
+    private val behandlingRepository: BehandlingRepository,
+    private val fagsakRepository: FagsakRepository,
```

Comment:
Kan ikke se at fagsakrepository blir brukt i servicen her, kan den fjernes?
Fant den heller ikke i ba-sak, så var litt usikker på om den skulle bli brukt senere eller ikke 😛

Reply:
Skrev om koden litt fra slik den var i ba og da hadde jeg visst ikke bruk for den lenger 👍 Nice catch!